website/Contribution/CONTRIBUTING.rst

```diff
@@ -909,7 +909,7 @@ Models
 
 * In a Model attribute order should be
 
-  #. Private attributes (`_name`, `_description`, `_inherit`, ...)
+  #. Private attributes (`_name`, `_inherit`, `_description`, ...)
```

Comment:
I think otherwise, because I consider, semantically, model specification would be read as "Take model A and add inheritances B and C". Besides, as I commented in commit description, this is just to gather what we already usually do.

Reply:
You can look at OCA and see that most of the time,  `_inherit` is put before `_name` (when `_name` is used).

Reply:
I prefer `_name` first too. It seems to be also [what upstream does](https://github.com/search?q=repo%3Aodoo%2Fodoo+_name+_inherit+path%3A%2F%5Eaddons%5C%2F%2F+language%3APython&type=code&l=Python).

Reply:
That was my rasoning, too. Specially in new code, that's the preferred order.